{"ID": "c2533265-6628-48cb-9fb0-02561e87c7bb", "effective_cve": "CVE-2018-3078", "image": {"exact": "docker.io/anchore/test_images@sha256:f56164678054e5eb59ab838367373a49df723b324617b1ba6de775749d7f91d4"}, "label": "FP", "package": {"name": "mysql", "version": "2.18.1"}, "timestamp": "2022-09-14T17:47:12+01:00", "user": "westonsteimel", "vulnerability_id": "CVE-2018-3078"}